calcom/cal.com
# fix: teams are listed on insights but then you cant find them on settings page 11087 cal 2401

## What does this PR do?

- Changes prisma query to fetch teams where membership relationship doesn't exist.
- Add invalidates for trpc when adding new members to other teams.



Fixes #11087 

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? Orgs should be activated.
- What are the minimal test data to have? An org with 2 team members, 1 team with owner membership, other with no members and you should be logged in as owner of org.
- What is expected (happy path) to have (input and output)? Go to /settings/organizations/teams and you should have 1 team listed on My Teams and other team listed on Other teams.
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
